server/create_test.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/ollama/ollama/api"
+)
+
+func TestConvertFromSafetensorsPathValidation(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "safetensors-test")
+	if err != nil {
+		t.Fatalf("Failed to create temp dir: %v", err)
+	}
+	defer os.RemoveAll(tempDir)
+
+	testContent := []byte("test content")
+	testPath := filepath.Join(tempDir, "source.safetensors")
+	err = os.WriteFile(testPath, testContent, 0o644)
+	if err != nil {
+		t.Fatalf("Failed to write test file: %v", err)
+	}
+
+	tests := []struct {
```

Comment:
There only "invalid" cases here. What about valid? What do those look like and how do they compare to the invalid?